extensions/amp-sidebar/1.0/sidebar.jss.js

```diff
@@ -60,13 +63,21 @@ const defaultBackdropStyles = {
   backgroundColor: 'rgba(0, 0, 0, 0.5)',
 };
 
+//TODO(#32400): See PR description.  This is a workaround dependent on a
+// a browser bug fix and should be removed when the browser bug is fixed.
+const sidebarChildClass = {
```

Comment:
The name is not very indicative. This is only used for the backdrop? I'd call it something like "backdropOverscrollBlocker".

Nit: no need for "Class" suffix. These are used as `classes.sidebarChild`, so "Class" suffix is rather duplicative.

Reply:
Thanks, updated in the latest!